packages/gasket-plugin-nextjs/lib/index.js

```diff
@@ -255,14 +255,6 @@ module.exports = {
           parent: 'express',
           after: 'next'
         },
-        {
-          name: 'next',
```

Comment:
Good catches and thanks for being willing to contribute @Zeedinstein. As @jpage-godaddy mentioned, we would still need this entry for the graph generation.

So yea, a couple issues... we should probably only generated one or the other depending on which, either Express or Fastify, Gasket plugins are in use by an app when using the `gasket docs` command. However, that still will not address the duplication in the generated index readme for the repo here. I think the duplication will be necessary, with the best thing to do being to clarify things in the descriptions:
- `Update the Next.js app instance before preparing for Express`
- `Update the Next.js app instance before preparing for Fastify`